### PR TITLE
docs: update guides and public APIs for NgModules

### PR DESCRIPTION
**Pull request by jelbourn**
R: @kara @hansl

### CODE REVIEW

File: src/components/menu/README.md [hunk 1]
```diff
@@ -7,30 +7,17 @@
 
 ### Setup
 
-`md-menu` relies on the existence of an app-wide `OverlayContainer`. This is the container in which 
-it will display. You'll need to import this from `@angular2-material/core` and add it to your 
-providers list on bootstrap.
+Impor the MdMenu module.
```
Comment (shatgupt) [inline src/components/menu/README.md]: Typo: `Import`

File: GETTING_STARTED.md [hunk 2]
```diff
@@ -98,96 +100,43 @@ Now you should be able to import the components normally wherever you'd like to
...
-```
 
-- Import any and all forms symbols - `NgForm`, `Validators`, etc - from `@angular/forms`. 
-Importing them from `@angular/common` will result in a `No value accessor found` error.
+### Additional setup for `md-slide-toggle` and `md-slider`:
+The slide-toggle and slider components have a dependency on [HammerJS](http://hammerjs.github.io/).
+1) Add HammerJS to your application via [npm](https://www.npmjs.com/package/hammerjs), a CDN 
+   (such as the [Google CDN](https://developers.google.com/speed/libraries/#hammerjs)), 
+   or served directly from your app.
+2) Include the typings for HammerJS in your typescript build.
```
Comment (robertmesserle) [inline GETTING_STARTED.md]: Should we include more detailed steps on this for those not super familiar with typescript?

File: GETTING_STARTED.md [hunk 3]
```diff
@@ -98,96 +100,43 @@ Now you should be able to import the components normally wherever you'd like to
...
 
-- Update your form code to use the new APIs. See more information about the changes in the proposal 
-doc [here](https://docs.google.com/document/u/1/d/1RIezQqE4aEhBRmArIAS1mRIZtWFf6JxN_7B4meyWK0Y/pub) 
-and the official documentation [here](https://angular.io/docs/ts/latest/guide/forms.html).
 
-### Additional steps for `md-icon` setup:
+### Additional setup for `md-icon` setup:
```
Comment (robertmesserle) [inline GETTING_STARTED.md]: Additional setup for setup?

**Comment by jelbourn** [comment]
@robertmesserle comments addressed

**Pull request merged**